samples/sampler/stories/qa/Picker.js

```diff
@@ -443,18 +456,60 @@ export const ForIrregularNumbers = (args) => (
 );
 
 boolean('joined', ForIrregularNumbers, Picker);
+boolean('noIndicator', ForIrregularNumbers, Picker);
 select('orientation', ForIrregularNumbers, prop.orientation, Picker);
 
 ForIrregularNumbers.storyName = 'for irregular numbers';
 
-export const InPopupTabLayout = () => <PickerInPopupTabLayout />;
+const PickerInPopupTabLayout = ({args}) => {
+	return (
+		<>
+			<PopupTabLayout open index={0}>
+				<Tab title="Display">
+					<TabPanels>
+						<TabPanel>
+							<Header title="Display 0" type="compact">
+								<slotAfter>
+									<Button>hello</Button>
+								</slotAfter>
+							</Header>
+							<div>
+								<Picker
+									onChange={action('onChange')}
+									width={args['width']}
+									orientation={args['orientation']}
+									wrap={args['wrap']}
+									joined={args['joined']}
+									noIndicator={args['noIndicator']}
+									noAnimation={args['noAnimation']}
+									disabled={args['disabled']}
+									incrementIcon={args['incrementIcon']}
+									decrementIcon={args['decrementIcon']}
+								>
+									{pickerList.vegetables}
+								</Picker>
+							</div>
+						</TabPanel>
+					</TabPanels>
+				</Tab>
+			</PopupTabLayout>
+		</>
+	);
+};
+
+export const InPopupTabLayout = (args) => <PickerInPopupTabLayout args={args} />;
+
+select('width', InPopupTabLayout, prop.width, Picker, 'large');
```

Comment:
the default size was changed. any reason?

BEFORE : 
width={select('width', prop.width, Picker, 'small')}